src/users/components/UserList.vue

```diff
@@ -18,12 +18,28 @@
       <q-item-separator />
       <q-collapsible
         v-if="inactiveUsers.length > 0"
-        icon="fas fa-bed"
-        :label="$t('GROUP.INACTIVE')"
-        :sublabel="inactiveSublabel"
         @show="showInactive = true"
         @hide="showInactive = false"
       >
+        <template slot="header">
+          <q-item-side>
+            <q-item-tile icon="fas fa-bed" />
+          </q-item-side>
+          <q-item-main
+            :label="$t('GROUP.INACTIVE')"
+            :sublabel="inactiveSublabel"
+          />
+          <q-item-side>
```

Comment:
On very small screens, this can hide the "Inactive" label. Maybe reduce the default margin/padding on mobile to make it less likely.

Reply:
I'm not sure I understand you correctly. You mean that the info button would hide the label? How small would that screen need to be..? o.0
I don't even know how to make that happen in mobile view of the browser, I couldn't find a device that small in the list... :sweat_smile:

Reply:
Depends on the language you chose, e.g. to hide the Polish word partially it needs 250 pixel. Maybe we just ignore it.